controllers/user.controller.js

```diff
@@ -36,7 +36,8 @@ exports.register = async (req, res) => {
     body.password = bcrypt.hashSync(password, 10);
     const created = await user.create(body);
     if (created) {
-      const tokenParams = { id: created.id, email };
+      const { id, email } = created;
+      const tokenParams = { id, email, expiresIn: "2h" };
```

Comment:
why can't the repeated parameter expiresIn be transferred to the function itself?